src/app/backend/replicasetdetail.go

```diff
@@ -135,6 +135,27 @@ func GetReplicaSetDetail(client *client.Client, namespace string, name string) (
 	return replicaSetDetail, nil
 }
 
+// Deletes replica set with given name in given namespace and related pods
+func DeleteReplicaSetWithPods(client *client.Client, namespace string, name string) error {
```

Comment:
Add a todo that we need to do this in a transaction :) Otherwise, it may happen that we delete RC and not its pods.